assets/components/paymentFailureMessage/paymentFailureMessage.jsx

```diff
@@ -4,6 +4,7 @@
 
 import React, { type Node } from 'react';
 import { appropriateErrorMessage, type CheckoutFailureReason } from 'helpers/checkoutErrors';
+import { classNameWithModifiers } from 'helpers/utilities';
 import SvgExclamationAlternate from '../svgs/exclamationAlternate';
```

Comment:
Just a small detail, I think this import could be re-written as `import SvgExclamationAlternate from 'svgs/exclamationAlternate';` to show that comes from the top `svgs` folder.

Reply:
I get a cannot resolve module error when I try this?